shoop/front/apps/auth/forms.py

```diff
@@ -30,6 +30,11 @@ def clean_username(self):
         if '@' in username:
             try:
                 username = user_model.objects.get(email=username).username
+            except ObjectDoesNotExist:
+                raise forms.ValidationError(
+                    _("Email and password didn't match, please try again."),
```

Comment:
This should be the same error message (and code) that `AuthenticationForm` itself uses when authentication fails. Otherwise an attacker could use the error message to deduce whether or not the email they entered actually exists in the database.